docs/core/compatibility/10.0.md

```diff
@@ -16,6 +16,12 @@ If you're migrating an app to .NET 10, the breaking changes listed here might af
 >
 > This article is a work in progress. It's not a complete list of breaking changes in .NET 10. To query breaking changes that are still pending publication, see [Issues of .NET](https://issuesof.net/?q=%20is:open%20-label:Documented%20is:issue%20(label:%22Breaking%20Change%22%20or%20label:breaking-change)%20(repo:dotnet/docs%20or%20repo:aspnet/Announcements)%20group:repo%20(label:%22:checkered_flag:%20Release:%20.NET%2010%22%20or%20label:10.0.0)%20sort:created-desc).
 
+## Containers
+
+| Title                                                                                                                      | Type of change      | Introduced version |
+|----------------------------------------------------------------------------------------------------------------------------|---------------------|--------------------|
+| [Default .NET images now using Ubuntu](containers/10.0/default-images-use-ubuntu.md)                                       | Behavioral change   | Preview 1          |
```

Comment:
nit: I feel like `now using` and `now use` are being used inconsistently. For example, I think the usage on the line that is commented should be `now use` since there is no verb (e.g. `are`). If it had a verb like `are`, it would make sense to be `are now using`. In general, if it there is no verb, then use `now use` but if there is a verb, such as `are`, then use `are now using`.

Is the word `now` necessary. Isn't it understood that the breaking change is a point-in-time change, so it's already implied that "now" is at the time of the breaking change?